internal/rpchelp/helpdescs_en_US.go

```diff
@@ -43,6 +43,15 @@ var helpDescsEnUS = map[string]string{
 	"createmultisigresult-address":      "The generated pay-to-script-hash address",
 	"createmultisigresult-redeemScript": "The script required to redeem outputs paid to the multisig address",
 
+	// CreateVotingAccountCmd help.
+	"createvotingaccount--synopsis":  "Create a voting account with account name and xpub.",
+	"createvotingaccount-name":       "Voting account name",
+	"createvotingaccount-pubkey":     "MasterPubKey from voting wallet",
```

Comment:
Don't call this the master pubkey, it should instead be the account extended pubkey.